### ISSUE TITLE
Set up config for parse server

### ISSUE THREAD

**Issue by banphlet** [ParaffinIoT/apiserver#1]
#Issue Summary

Currently Parrafin parse server has no well defined config for handling environmental variable.
eg. ```javascript 
const parseSetting = {
...config
}``` defined in index.js.

**Comment by expandboard** [comment]
Certainly, config file can handle environment variables better. putting secrets in config file needs encryption. do you have any suggestion for encryption?

**Comment by banphlet** [comment]
@expandboard  env themselves would be encrypted by ci or set only on the server like how heroku handles .envs

**Comment by banphlet** [comment]
ok
